core/node/state_keeper/src/io/seal_logic/mod.rs

```diff
@@ -358,6 +361,17 @@ impl L2BlockSealCommand {
         // Run sub-tasks in parallel.
         L2BlockSealProcess::run_subtasks(self, strategy).await?;
 
+        let progress = L2_BLOCK_METRICS.start(L2BlockSealStage::CalculateLogsBloom, is_fictive);
+        let iter = self.l2_block.events.iter().flat_map(|event| {
+            event
+                .indexed_topics
+                .iter()
+                .map(|topic| BloomInput::Raw(topic.as_bytes()))
+                .chain(std::iter::once(BloomInput::Raw(event.address.as_bytes())))
```

Comment:
Nit: IIUC, you can use `[BloomInput::Raw(_)]` (i.e., a single-item array) instead of `iter::once()` for brevity.